.github/workflows/node-ci.yml

```diff
@@ -5,14 +5,18 @@ on: push
 jobs:
   test:
 
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
+
+    strategy:
+      matrix:
```

Comment:
I'm curious, why use `matrix` if there's only one version?

Reply:
copy and paste from historic code more than anything else. I think previously it was testing against multiple versions but I don't really feel thats necessary; for front ends it seems most appropriate to build at our target docker version which in turn should ideally be latest node LTS